ipwb/assets/serviceWorker.js

```diff
@@ -29,7 +29,7 @@ const rc = new Reconstructive({
 // <   localResource: function(FetchEvent) => boolean
 // < }
 rc.exclusions.specialEndpint = function (event, config) {
-  return ['/ipwbassets/', '/daemon/', '/config/'].some(
+  return ['/ipwbassets/', '/ipfsdaemon/', '/config/'].some(
```

Comment:
Should we also change `/config` to `/ipwbconfig`?

Reply:
`/config/` updated in 26436f9.